spec/namespaces/ism.yaml

```diff
@@ -70,6 +134,20 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/ism.remove_policy@200'
+  /_plugins/_ism/change_policy:
+    post:
+      operationId: ism.change.0
+      x-operation-group: ism.change
```

Comment:
This should be in the same group as `ism_change_policy`

Reply:
But they have different parameters, one specifies the index in the query, the other in the path. So if you try to make them the same group you get this:

```
{
  file: 'namespaces/ism.yaml',
  location: 'Operation Group: ism.change_policy',
  message: "2 'ism.change_policy' operations must have an identical set of query parameters."
}
```

Reply:
Updated.